src/script/calling/CallingRepository.ts

```diff
@@ -363,8 +363,16 @@ export class CallingRepository {
     }
   }
 
+  /**
+   * Checks if browser supports all WebRTC APIs which are required for conference calling. Users with Chrome 83 need to enable "Experimental Web Platform features" (--enable-experimental-web-platform-features) to use all required APIs.
```

Comment:
maybe breaking this line into multiple ones for easier reading.

Reply:
The linter didn't complain. So I will keep it like this.